### PR TITLE
Add nodeSelector to fields copied with use-agent-node-affinity

### PR DESCRIPTION
**Pull request by keith**
This field can be used as a simpler version of affinity

### CODE REVIEW

File: hooks/command [hunk 1]
```diff
@@ -89,9 +89,10 @@ job_spec="$(jsonnet \
   "${basedir}/lib/job.jsonnet")"
 
 if [[ "$use_agent_node_affinity" == "true" ]]; then
-  for field in affinity tolerations; do
-    buildkite_agent_value="$(kubectl get pod "$(cat /etc/hostname)" -o json | jq ".spec.$field")"
-    job_spec="$(echo "$job_spec" | jq ".spec.template.spec.$field=$buildkite_agent_value")"
+  agent_spec="$(kubectl get pod "$(cat /etc/hostname)" -o json)"
+  for field in affinity tolerations nodeSelector; do
```
Comment (iffyio) [inline hooks/command:93]: Let's update the doc entry to mention that this copies nodeSelector as well https://github.com/EmbarkStudios/k8s-buildkite-plugin#use-agent-node-affinity-optional-boolean

**Review by iffyio — COMMENTED** [review]
Left a comment to update the docs otherwise this looks good to me! Thanks :)

File: hooks/command (hunk 1 above)
Comment (keith) [reply hooks/command:93]: Done!

**Review by iffyio — APPROVED** [review]
Thanks!

**Pull request merged**